server/server.py

```diff
@@ -180,7 +180,11 @@ def submit(sid, data):
   print("answer", sid, data)
   puzzle, answer = [data.get(key) for key in ['puzzle', 'answer']]
   # TODO: error gracefully if no puzzle or answer
-  correct = answer.lower() == ANSWERS.get(str(puzzle))
+  if type(answer) is list: 
```

Comment:
nit: could be more explicit here and say `if type(answer) is string:`, then use `answer.lower()`, in case in the future we send dictionaries as an answer and we'd run into the same error

Reply:
I get a NameError when I put string.

"NameError: global name 'string' is not defined"

Reply:
used isinstance() instead

Reply:
ah i think you have to use str? isinstance works too!